samples/graph-activity-feed/csharp/ActivityFeedNotification/Helpers/TeamMembersNotificationRecipient.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Newtonsoft.Json;
+
+namespace Microsoft.Graph
+{
+    //
+    // Summary:
+    //     The type TeamMembersNotificationRecipient.
+    [JsonObject(MemberSerialization = MemberSerialization.OptIn)]
+    public class TeamMembersNotificationRecipient : TeamworkNotificationRecipient
+    {
+        //
+        // Summary:
+        //     Gets or sets teamId.
```

Comment:
This summary alignment doesn't looks correct.

Reply:
Done